### PR TITLE
569 automated test data refresh

### PR DESCRIPTION
**Pull request by joeltejeda**
Ref #569

### CODE REVIEW

File: tests/browser-automated-tests-playwright/tests/test_order_plan_with_only_recurring_charge.js [hunk 1]
```diff
@@ -83,17 +83,24 @@ async function test_order_plan_with_only_recurring_charge(browsers, browserConte
 
     const subscriber_plan_sell_price_content = await page.evaluate(() => document.querySelector('.subscribers-plan-sell-price').textContent.indexOf("(No up-front fee)"));
     assert(subscriber_plan_sell_price_content > -1)
+    
+    // Click Refresh Subscription
+    await page.click('#refresh_subscriptions'); // this is the refresh subscription
+    await page.textContent('.alert-heading') === "Notification";
```
Comment (joeltejeda) [inline tests/browser-automated-tests-playwright/tests/test_order_plan_with_only_recurring_charge.js:89]: I changed the variable for this line

File: tests/browser-automated-tests-playwright/tests/test_order_plan_with_only_upfront_charge.js [hunk 2]
```diff
@@ -78,6 +78,13 @@ async function test_order_plan_with_only_upfront_charge(browsers, browserContext
     // Verify that plan is attached to subscriber
     const subscriber_plan_title_content = await page.textContent('.subscription-title');
     assert(subscriber_plan_title_content === 'One-Off Soaps');
+   
+   // Click Refresh Subscription
+    await page.click('#refresh_subscriptions'); // this is the refresh subscription
+    await page.textContent('.alert-heading') === "Notification";
```
Comment (chrisjsimpson) [inline tests/browser-automated-tests-playwright/tests/test_order_plan_with_only_upfront_charge.js:84]: Same , but checking for different price https://github.com/Subscribie/subscribie/pull/573/files#r643543510

File: tests/browser-automated-tests-playwright/tests/test_order_plan_with_subscription_and_upfront_charge.js [hunk 3]
```diff
@@ -70,14 +70,20 @@ async function test_order_plan_with_subscription_and_upfront_charge(browsers, br
...
     const subscriber_email_content = await page.textContent('.subscriber-email');
     assert(subscriber_email_content === 'john@example.com');
 
     // Verify that plan is attached to subscriber
     const subscriber_subscription_title_content = await page.textContent('.subscription-title');
     assert(subscriber_subscription_title_content === 'Hair Gel');
+    
+    // Click Refresh Subscription
+    await page.click('#refresh_subscriptions'); // this is the refresh subscription
+    await page.textContent('.alert-heading') === "Notification";
```
Comment (chrisjsimpson) [inline tests/browser-automated-tests-playwright/tests/test_order_plan_with_subscription_and_upfront_charge.js:83]: Same , but checking for both up-front and recurring price https://github.com/Subscribie/subscribie/pull/573/files#r643543510

File: tests/browser-automated-tests-playwright/tests/test_order_plan_with_only_recurring_charge.js [hunk 4]
```diff
@@ -83,17 +83,24 @@ async function test_order_plan_with_only_recurring_charge(browsers, browserConte
...
+    // screeshot to the active subscriber
```
Comment (chrisjsimpson) [inline tests/browser-automated-tests-playwright/tests/test_order_plan_with_only_recurring_charge.js:90]: Good commenting thanks

**Review by chrisjsimpson — APPROVED** [review]
Thanks @joeltejeda :confetti_ball:

**Pull request merged**